zcash_proofs/src/prover.rs

```diff
@@ -95,6 +101,23 @@ impl LocalTxProver {
 
         Some(LocalTxProver::new(&spend_path, &output_path))
     }
+
+    /// Creates a `LocalTxProver` using Sapling parameters bundled inside the binary.
+    ///
+    /// This requires the `bundled-prover` feature, which will increase the binary size by
+    /// around 50 MiB.
+    #[cfg(feature = "bundled-prover")]
+    pub fn bundled() -> Self {
+        let (spend_buf, output_buf) = wagyu_zcash_parameters::load_sapling_parameters();
+        let (spend_params, spend_vk, output_params, _, _) =
```

Comment:
This pattern match requiring you to get things in the correct order seems sketchy, as mentioned above.